games-strategy/freeciv/freeciv-2.5.10.recipe

```diff
@@ -0,0 +1,119 @@
+SUMMARY="A free Civilization-like game"
+DESCRIPTION="Freeciv is a free Civilization-like game, primarily \
+for X under Unix. It has support for multiplayer games locally or \
+over a network, and an AI which gives most people a run for their money.
+Freeciv aims to be mostly rule-compatible with Civilization II [tm], \
+published by Sid Meier and Microprose [tm]. A few rules are different \
+where we think it makes more sense, and we have lots and lots of \
+adjustable parameters to make customizing games possible.
+Freeciv has been implemented completely independently of Civilization; \
+you do not need to own Civilization to play Freeciv."
+HOMEPAGE="http://www.freeciv.org/"
+COPYRIGHT="1996-2018 The Freeciv Team"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://downloads.sourceforge.net/freeciv/freeciv-$portVersion.tar.bz2"
+CHECKSUM_SHA256="c8a14156562dce2f0a7b96390c2d5a35314f1124c448782a6a47b0c51cfcb302"
+PATCHES="freeciv-$portVersion.patchset"
+ADDITIONAL_FILES="freeciv.rdef.in"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+GLOBAL_WRITABLE_FILES="
+	settings/freeciv/database.lua keep-old
+	"
```

Comment:
Looks like we also need an entry for this in the [Guidelines][0]. I see different recipes place this in different places

[0]: https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines

Reply:
Probably needs to move up before ARCHITECTURES ...

Reply:
I just did some metrics, and you are right :)

Total number of recipes having a `GLOBAL_WRITABLE_FILES` | 147
--- | ---
Recipes with `GLOBAL_WRITABLE_FILES` **_before_** `PROVIDES` | 116
Rrecipes with `GLOBAL_WRITABLE_FILES` **_after_** `PROVIDES` | 31

Reply:
Directories with recipes having both `GLOBAL_WRITABLE_FILES` and `USER_SETTINGS_FILES` | 15 | Names
--- | --- | ---
Recipes with `GLOBAL_WRITABLE_FILES` **_before_** `USER_SETTINGS_FILES`| **10** | bash_completion ccache dovecot geoipupdate johntheripper lighttpd lynx neofetch openldap openssh privoxy
Recipes with `GLOBAL_WRITABLE_FILES` _after_ `USER_SETTINGS_FILES` | 5 | git p11-kit softshm tor wget

Reply:
Looks like we need a guideline on where to put these then :)

Reply:
I've updated the [wiki](https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines) :)
(See my [message in PR 2135](https://github.com/haikuports/haikuports/issues/2135#issuecomment-361050885).)